cypress/support/pages/yves/ssp-dashboard-management/repositories/suite-ssp-dashboard-management-repository.ts

```diff
@@ -0,0 +1,79 @@
+import { injectable } from 'inversify';
+import { SspDashboardManagementRepository } from '../ssp-dashboard-management-repository';
+import Chainable = Cypress.Chainable;
+
+@injectable()
+export class SuiteSspDashboardManagementRepository implements SspDashboardManagementRepository {
+  getUserInfoBlock(): Chainable {
+    return cy.get('div[data-qa="user-info"]');
```

Comment:
Good that we use data-qa!
could you please remove all 'div' from all lotators, so we rely exactly on `[data-qa="` no matter what it would be (div, strong, li ...)